images/dind/node/openshift-generate-node-config.sh

```diff
@@ -8,6 +8,14 @@ source /usr/local/bin/openshift-dind-lib.sh
 # Should set OPENSHIFT_NETWORK_PLUGIN
 source /data/network-plugin
 
+function lock() {
```

Comment:
We probably want to namespace this function, I feel like we could have name collisions with `lock`